CVLByExample/TransientStorage/certora/specs/Mutexer.spec

```diff
@@ -0,0 +1,44 @@
+methods {
+
+    function lockValue() external returns (uint256) envfree;
+    function getContractLock() external returns (uint256) envfree;
+}
+
+definition isLocked(uint256 status) returns bool = status == lockValue();
+
+persistent ghost bool contract_lock_status;
+
+hook ALL_TSTORE(uint loc, uint v) {
+    if (loc == getContractLock()) {
```

Comment:
can we add 'executingContract' to this example? or do we already have an example that uses executingContract?